app/src/app/home/pages/team/page.component.ts

```diff
@@ -154,10 +154,10 @@ export class TeamPageComponent
           this.router.navigate(['~', activeTeam]);
         }
         this.data.teams = v;
+        this.processToken();
       }),
       team: teamPageService.data.team$.subscribe((v) => {
         this.data.team = v;
-        this.processToken();
       })
```

Comment:
Unrelated bugfix. If user have 0 teams, they should still be able to perform `touca login`.